### DISCUSSION TITLE
v2.0.0

### DISCUSSION THREAD

**Discussion by deleteLater** [featbit/featbit#306] [Announcements]
## What's Changed
* ✨ feat: (api) add feature flag description by @DaveFz in https://github.com/featbit/featbit/pull/258
* ✨ feat: (ui) add description field to new feature form by @sagar110599 in https://github.com/featbit/featbit/pull/259
* 📖 doc: update contribution task template by @cosmos-explorer in https://github.com/featbit/featbit/pull/263
* 🧹 refactor: add missing translations by @cosmos-explorer in https://github.com/featbit/featbit/pull/264
* 🐛 fix: fixed flag trigger url not correct bug by @cosmos-explorer in https://github.com/featbit/featbit/pull/266
* ✨ feat: added a tooltip for trigger url to tell how to invoke a trigger by @cosmos-explorer in https://github.com/featbit/featbit/pull/267
* ✨ feat: added description field to feature flag setting page by @cosmos-explorer in https://github.com/featbit/featbit/pull/270
* ✨ feat: implemented a feature to allow users to send feedback to featbit maintainer by @cosmos-explorer in https://github.com/featbit/featbit/pull/271
* 📖 doc: fixed google sheet link by @cosmos-explorer in https://github.com/featbit/featbit/pull/272
* 📖 doc: swap getting started and features section by @cosmic-flood in https://github.com/featbit/featbit/pull/274
* ✨ feat: add environment key by @cosmos-explorer in https://github.com/featbit/featbit/pull/282
* 🐛 fix: api & evaluation-server shutdown errors by @deleteLater in https://github.com/featbit/featbit/pull/283
* 🧹 refactor: Improved email validation error message by @cosmos-explorer in https://github.com/featbit/featbit/pull/284
* ✨ feat: crud access tokens by @cosmos-explorer in https://github.com/featbit/featbit/pull/268
* ✨ feat: hide edit button for system policies by @san-ki in https://github.com/featbit/featbit/pull/286
* ✨ feat: implement permission based authorization for feature flag by @cosmos-explorer in https://github.com/featbit/featbit/pull/285
* 🧹 refactor: add policy type enum by @cosmos-explorer in https://github.com/featbit/featbit/pull/288
* ✨ feat: added button to copy environment Id by @cosmos-explorer in https://github.com/featbit/featbit/pull/290
* ✨ feat: add api to retrieve the latest data for the server and client sdks by @deleteLater in https://github.com/featbit/featbit/pull/275
* 📖 doc: added contributor list into README by @cosmic-flood in https://github.com/featbit/featbit/pull/293
* ✨ feat: auditLog supports changes to feature flag description by @kabir-webDev in https://github.com/featbit/featbit/pull/291
* 📖 doc: refactor README.md change auto-generated by all-contributors bot by @cosmic-flood in https://github.com/featbit/featbit/pull/303
* 🧹 refactor: regenerate translations by @cosmos-explorer in https://github.com/featbit/featbit/pull/304
* 📖 doc: added step to contribution task template by @cosmos-explorer in https://github.com/featbit/featbit/pull/305
* ✨ feat: added orgId to request header by @cosmos-explorer in https://github.com/featbit/featbit/pull/289

## New Contributors
* @DaveFz made their first contribution in https://github.com/featbit/featbit/pull/258
* @sagar110599 made their first contribution in https://github.com/featbit/featbit/pull/259
* @san-ki made their first contribution in https://github.com/featbit/featbit/pull/286
* @kabir-webDev made their first contribution in https://github.com/featbit/featbit/pull/291

## Note
If you’re using version 1.x, please note that version 2.x is not compatible. If you’d like to upgrade to version 2.x, please [contact us](https://join.slack.com/t/featbit/shared_invite/zt-1ew5e2vbb-x6Apan1xZOaYMnFzqZkGNQ) for assistance.

**Full Changelog**: https://github.com/featbit/featbit/compare/1.4.1...2.0.0

<hr /><em>This discussion was created from the release <a href='https://github.com/featbit/featbit/releases/tag/2.0.0'>v2.0.0</a>.</em>

**Comment by cosmic-flood** [comment]
You didn't mention Go and .NET SDK in the 2.0.0 👯 
You didn't mention Benchmark report either 🔢

**Reply by deleteLater** [reply]
This discussion was auto-generated from the main repository release and contains only the **code changes**.
